lib/libutils/ext/ubsan.c

```diff
@@ -188,20 +217,25 @@ void __ubsan_handle_out_of_bounds(void *data_, void *idx __unused)
 	ubsan_handle_error(__func__, &data->loc, ubsan_panic);
 }
 
-void __ubsan_handle_builtin_unreachable(void *data_)
+/*
+ * __ubsan_handle_builtin_unreachable and __ubsan_handle_missing_return
+ * will give "‘noreturn’ function does return" warning, because
```

Comment:
Warnings aren't accepted, you need to define `ubsan_panic()` so that it doesn't cause warnings.

Reply:
Done